graylog2-server/src/main/resources/org/graylog2/featureflag/feature-flag.config

```
@@ -92,7 +92,7 @@ remote_reindex_migration=off
 data_warehouse_search=off
 
 # Enable preview of input setup wizard
-setup_mode=off
+setup_mode=on
```

```suggestion
setup_mode=on
```

Should be off by default